.github/workflows/oracle-workflow-test.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and Test Oracle Connector
+
+on:
+  push:
+    branches:
+      - main
+      - 1.*
+      - oracle-ci-test
+    paths:
+      - 'support/checkstyle/**'
+      - 'debezium-api/**'
+      - 'debezium-ddl-parser/**'
+      - 'debezium-assembly-descriptors/**'
+      - 'debezium-core/**'
+      - 'debezium-embedded/**'
+      - 'debezium-connector-oracle/**'
+      - 'debezium-parent/pom.xml'
+      - 'debezium-bom/pom.xml'
+      - 'pom.xml'
+      - '.github/workflows/oracle-workflow.yml'
+      - '.github/workflows/oracle-workflow-test.yml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup java 17
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: 17
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Check changes in Debezium Connector Oracle LogMiner
+        env:
+          QUAY_IO_USERNAME: ${{ secrets.QUAY_IO_USERNAME }}
+          QUAY_IO_PASSWORD: ${{ secrets.QUAY_IO_PASSWORD }}
```

Comment:
I know there must be legal considerations regarding running tests on a public CI platform with Oracle but using a private image means that contributors won't be able to use it for local builds.

Would it be possible to document how this image is built or instead maybe use one of the publicly available images?

Reply:
@morozov We will definitely publish the Dockerfiles to allow you building local images

Reply:
FWIW, these images are built using the same process outlined in:
https://github.com/debezium/oracle-vagrant-box

The only key difference here is the database files are incorporated as a part of the image rather than a volume as outlined in the above link mostly so that we can throw away and redeploy with no concern, but that's a technical detail that is mentioned in the vagrant box documentation.

Lastly, the Dockerfile's used for building these images are already available, see:
https://github.com/oracle/docker-images/tree/main/OracleDatabase/SingleInstance/dockerfiles

These are used by the Vagrant box documentation.

Reply:
Hi @morozov the Oracle XE versions are not able to run Debezium.  Oracle specifically prohibits the use of Supplemental Logging and several other core features that are necessary to do any type of change data capture.

Reply:
Oh, I see. Thanks for the explanation.